common/supplemental/supplementalData.xml

```diff
@@ -1454,12 +1454,16 @@ XXX Code for transations where no currency is involved
 		<language type="cjm" scripts="Cham"/>
 		<language type="cjm" scripts="Arab" alt="secondary"/>
 		<language type="cjs" scripts="Cyrl"/>
-		<language type="ckb" scripts="Arab" territories="IQ IR"/>
+		<language type="ckb" scripts="Arab"/>
+		<language type="ckb" territories="IQ IR" alt="secondary"/>
```

Comment:
Until we fix [CLDR-14992](https://unicode-org.atlassian.net/browse/CLDR-14992), we need to manually keep the existing line 1457, which was changed as a temporary way to address [CLDR-13595](https://unicode-org.atlassian.net/browse/CLDR-13595.).

Reply:
I'll do that and add a comment.

Reply:
Still seems to be wrong here...

Reply:
i think you might have reviewed just before the push https://github.com/unicode-org/cldr/pull/1767/files#diff-5ea2936f709d9dc83508c70c9a589a5bdaadf9392a0ca96d9b56507621f4090bR1457

Reply:
Ah yes, looks OK now.